guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc

```diff
@@ -52,14 +52,25 @@ If you want to change the default values in the template, clone the template and
 If there is no PXELinux entry, or you cannot find the new template, navigate to *Hosts* > *All Hosts*, and on your host, click *Edit*.
 Click the *Operating system* tab and click the Provisioning Template *Resolve* button to refresh the list of templates.
 . Configure the `dhcpd.conf` file as follows:
```

Comment:
Like in the other document, this can be removed
```suggestion
```

Reply:
Addressed :+1: